src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,30 @@
 package core.basesyntax.service;
 
+import core.basesyntax.db.Storage;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
 
+    private static final int MINIMAL_PASSWORD_LENGTH = 6;
+    private static final int MINIMAL_AGE = 18;
+
     @Override
     public User register(User user) {
-        return null;
+        if (user.getLogin() == null || user.getLogin().isEmpty()) {
+            throw new RuntimeException();
+        }
+        if (user.getPassword() == null || user.getPassword().length() < MINIMAL_PASSWORD_LENGTH) {
+            throw new RuntimeException();
+        }
+        if (user.getAge() == null || user.getAge() < MINIMAL_AGE) {
+            throw new RuntimeException();
```

Comment:
```suggestion
            throw new RuntimeException("The user does not meet the minimum age requirement");
```
Давай додамо більше кастомних повідомлень про помилку